tests/operator/resourcefiles/templates/vm-template.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: kubevirt.io/{{ .Version }}
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-{{ .Version }}
+  name: vm-{{ .Version }}
+spec:
+  dataVolumeTemplates:
+    - metadata:
+        name: test-dv{{ .Index }}
+      spec:
+        pvc:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+        source:
+          blank: {}
+  runStrategy: Manual
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-{{ .Version }}
+    spec:
+      domain:
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: containerdisk
+            - disk:
+                bus: virtio
+              name: cloudinitdisk
+            - disk:
+                bus: virtio
+              name: datavolumedisk1
```

Comment:
I think the name here is an oversight that existed prior to this PR; probably `datavolumedisk{{ .Index }}` is what it we were going for. I don't see any functional implication though

Reply:
Not sure. This is the internal name within the VMI, rather than the underline data source (lines 45-47). Maybe we can drop the number (`1`) . But using the index here may be confusing.

Reply:
Dropping the index would work yes

Reply:
fixed. Thanks @akalenyu !

BTW - this is a great example how moving the yaml string to a dedicated file, makes it easier to review.